docs/frb-flutter-migration.md

```
@@ -0,0 +1,204 @@
+                                                                  ▼
+                                                  ┌──────────────────────────┐
+                                                  │ marmot-protocol/whitenoise│
+                                                  │  pubspec.yaml git dep on │
+                                                  │  flutter-package branch  │
+                                                  └──────────────────────────┘
+```
+
+## Where things live
+
+### On `master` (Rust-only)
+| Concern | Path |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)